### PR TITLE
Updated link and page heading in the cross-reference

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Updated prod doc link to alias and changed page heading in the cross-ref
- **Is this PR related to a Monolith release?**No 

### Resolves:

* [OKTA-396431](https://oktainc.atlassian.net/browse/OKTA-396431)

### CODE REVIEW

**Review by susanharper-okta — APPROVED** [review]
lgtm 🚢

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/register-app-in-okta/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ If you need to create a new OIDC app integration:
...
     > **Note:** The Authorization Code grant flow is also supported.
-1. On the **Assignments** tab, click **Assign** to assign the app integration to any user or group in your org. Click **Done** when the assignments are complete. For instructions on how to assign the app integration to individual users and groups, see the [Assign applications for people and groups](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
+1. On the **Assignments** tab, click **Assign** to assign the app integration to any user or group in your org. Click **Done** when the assignments are complete. For instructions on how to assign the app integration to individual users and groups, see the [Assign app applications](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/register-app-in-okta/index.md:29]: Should be `[Assign app integrations]`

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-app/index.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Begin by signing up for an [Okta developer account](https://developer.okta.com/s
     Fill in this information and click **Test API Credentials** to test whether the Okta integration can connect to your SCIM API.
 
     Click **Save** to complete the API integration.
-1. On the **Assignments** tab, ensure that the right users and groups in your org are assigned to the app integration. For instructions on how to assign the app integration to individual users and groups, see the [Assign applications for people and groups](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
+1. On the **Assignments** tab, ensure that the right users and groups in your org are assigned to the app integration. For instructions on how to assign the app integration to individual users and groups, see the [Assign app integrations](hhttps://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-app/index.md:26]: Link fail - it has an extra 'h' at the start of the `https`

**Review by IanHakes-okta — CHANGES_REQUESTED** [review]
Two fixes to make - otherwise LGTM

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/register-app-in-okta/index.md (hunk 1 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/register-app-in-okta/index.md:29]: Fixed.

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-app/index.md (hunk 2 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-app/index.md:26]: Fixed

**Review by IanHakes-okta — APPROVED** [review]
LGTM!

**Pull request merged**